sdjournal/journal.go

```diff
@@ -220,6 +220,22 @@ func (j *Journal) GetData(field string) (string, error) {
 	return msg, nil
 }
 
+// SetDataThresold sets the data field size threshold for data returned by
+// GetData. To retrieve the complete data fields this threshold should be
+// turned off by setting it to 0, so that the library always returns the
+// complete data objects.
+func (j *Journal) SetDataThreshold(threshold uint64) error {
+	j.mu.Lock()
+	r := C.sd_journal_set_data_threshold(j.cjournal, C.size_t(threshold))
+	j.mu.Unlock()
+
+	if r != 0 {
```

Comment:
Everywhere else we're testing the return value from sd-journal using `if r < 0 {`, I see no reason to deviate here.